src/Domain.jl

```diff
@@ -958,10 +958,10 @@ end
     V = d.V(t)
     ns = y[d.indexes[1]:d.indexes[2]]
     T = y[d.indexes[3]]
-    N = sum(ns)
```

Comment:
Lets squash all of these "Use thermo variables to calculate N in calcthermo" commits